github/resource_github_organization_webhook.go

```diff
@@ -219,3 +219,49 @@ func resourceGithubOrganizationWebhookDelete(d *schema.ResourceData, meta interf
 	_, err = client.Organizations.DeleteHook(ctx, orgName, hookID)
 	return err
 }
+
+func webhookConfigFromInterface(config map[string]interface{}) *github.HookConfig {
```

Comment:
Ahh, is this workaround for fixing the bug I was complaining about [here](https://github.com/integrations/terraform-provider-github/pull/2188#issuecomment-2187446020)? Nice!

Reply:
Yes, this was the nicest way to not break backwards compatibility